styles/stylesheet.css

```diff
@@ -0,0 +1,407 @@
+*{
+  box-sizing: border-box;
+}
+
+/* ***Colors used in styling site:***
+Green color used in header and for CTAs (#00cc00)
+White font color (#ffffff)
+Blue font used in navigation and headers (#00394d)
+Light grey used for nav bar and borders (#e6e6e6)
+ */
+
+/* Setting the body text to be sans-serif, will use specific class styling as required for serif text */
+body {
+  font-family: 'Open Sans', 'Lato', sans-serif;
+}
+
+/* Removing the underlines from ALL links */
+a {
+  text-decoration: none;
+}
+
+/* Styling the header for images, fonts, etc. */
+header {
+  height: 30rem;
+  background-image: url("../images/header-image.jpg");
+  background-size: cover;
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 5% 5%;
+}
+
+.header-text {
+  color: #00cc00;
+  font-family: "Times New Roman", Times, serif;
+  font-style: italic;
+}
+
+header a {
+  color: #ffffff;
+}
+
+.cart {
+  width: 2rem;
+}
+
+.checkout {
+  background-color: #00cc00;
+  font-size: .75rem;
+  font-weight: 700;
+  padding: .5rem 2rem;
+}
+
+/*Styling for navigation*/
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 1rem;
+  overflow: hidden;
+  background-color: #e6e6e6;
+  font-family: "Times New Roman", Times, serif;
+  font-size: 1.5rem;
+}
+
+.navigation {
+  float: left;
+}
+
+.navigation a {
+  color: #00394d;
+  font-weight: 700;
+  text-align: center;
+  padding: 1.5rem 1.5rem;
```

Comment:
If you're using the same values for all four sides, you could even short hand this more

```
padding: 1.5rem
```

which will give you 1.5rem on all four sides.